src/utils/eventEmitter.ts

```diff
@@ -0,0 +1,53 @@
+import events from 'events';
+
+/**
+ * EventEmitter wrapper, where the events are defined by the given interface.
+ * Each property on the given interface is an event name, and the value should
+ * be a function where the arguments are the event parameters, and the return
+ * value is the return value of the event (most likely void).
+ */
+export default interface EventEmitter<T extends { [P in keyof T]: (...args: any) => void}> {
```

Comment:
This is somewhat hard to read -- I find it's better with a blank line after each definition (except the last)

Same for all the definitions of `spawnFile` in `utils/childProcess.ts`, but that ship sailed some time ago.